lang/ru.xml

```diff
@@ -349,14 +520,20 @@ depending on its size and your hardware.</app_delete_note>
 						<lang id="1024">Русский</lang>
 						<lang id="2048">Финский</lang>
 						<lang id="4096">Шведский</lang>
-						<lang id="524288">турецкий</lang>
+						<lang id="524288">Турецкий</lang>
```

Comment:
OMG. It's obvious mistake, but I found same mistake in the original vita firmware. So revert. :(